scripts/python_tests.sh

```diff
@@ -31,5 +31,4 @@ superset db upgrade
 superset init
 
 echo "Running tests"
-
-pytest --durations-min=2 --maxfail=1 --cov-report= --cov=superset ./tests/integration_tests "$@"
+pytest --durations-min=2 --maxfail=1 --cov-report=html --cov=superset --numprocesses=auto --random-order ./tests/integration_tests "$@"
```

Comment:
What's the benefit of the random order?

Reply:
Having the tests run in a deterministic order is a requirement given these terminate (for right or wrong) after the first failure and thus when debugging tests you need to ensure that the order is preserved between successive runs.